proto/subscription/v1beta1/subscription.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package subscription.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "subscription/v1beta1/cronspec.proto";
+
+option go_package = "github.com/crypto-com/chain-main/x/subscription/types";
+
+message Params {
```

Comment:
Some documentation on what these params mean. Or link to specs?

Reply:
done